mono/utils/mono-state.c

```diff
@@ -33,6 +33,17 @@ extern GCStats mono_gc_stats;
 #ifdef TARGET_OSX
 #include <mach/mach.h>
 #include <mach/task_info.h>
+
+// OSX 10.9 does not have MAP_ANONYMOUS
+#if !defined(
+  #define NO_MAP_ANONYMOUS
```

Comment:
This doesn't look right.

Reply:
Oops. Copy+paste error. I did test this change in my custom mono compilation on OSX 10.9